README.md

```diff
@@ -1,2 +1,45 @@
-# HW01
-Edit the readme
+---
+output:
+  html_document: default
+  pdf_document: default
+---
+# My life in Quarantine
+### Althea Bock-Hughes
+
+## Quarantine projects included:
+
+* Knitting
+
+* Baking Sourdough Bread
+
+* Watching "Avatar the Last Airbender" for the first time 
+
+* Tanning 
+
+* Occasionally thinking about science 
+
+My most involved project was **knitting** 
+I knit *four* sweaters in Quarantine
+
+1. This was the first one I knit and the hardest 
+
+![Sweater](IMG_0358.jpeg)
```

Comment:
I'm able to see the image from the link, but not from your desktop! I know why though! Essentially, you need to direct R studio to the image location. R Studio wouldn't be able to find IMG_0358.jpeg. If you type getwd() into the R console you will figure out where your working directory is. For me, it is /Users/nikitamehta/Documents. Then, I moved my photo to this folder, and typed in this command: ![Nikita Mehta](/Users/nikitamehta/Documents/Nikita.png). Basically the same thing you have, but with directions for how to get to "Nikita.png", my image file! I hope that helps :)